Data/extract_tags.py

```diff
@@ -0,0 +1,69 @@
+import argparse
+import csv
+import sys
+
+from lxml import etree as ET
+import tqdm
+
+
+def extract_annotations(xml_file, output_file):
+    """ Extract the annotations from pubtator xml formatted file
+    Outputs a TSV file with the following header terms:
+    Document - the corresponding pubmed id
+    Type - the type of term (i.e. Chemical, Disease, Gene etc.)
+    ID - the appropiate MESH or NCBI ID if known
+    Offset - the character position where the term starts
+    End - the character position where the term ends
+
+    Keywords arguments:
+    xml_file -- The path to the xml data file
+    output_file -- the path to output the formatted data
+    """
+    with open(output_file, "w") as csvfile:
+        fieldnames = ['Document', 'Type', 'ID', 'Offset', 'End']
+        writer = csv.DictWriter(csvfile, fieldnames=fieldnames)
```

Comment:
Also rename sample file to use a `.tsv` extension so GitHub does it's magic rendering.